### PR TITLE
Update CSwiftV.swift

### PR DESCRIPTION
**Pull request by cbarrerah**
Changes to enable swift 3 compatibility
Tried to keep the rest of the code untouched.

### CODE REVIEW

File: Sources/CSwiftV.swift [hunk 1]
```diff
@@ -50,29 +50,29 @@ public class CSwiftV {
...
+                let result = String(element.characters.dropFirst().dropLast())
+                return result
             }
             return element
         }
     }
 
     internal static func recordsFromString(string: String) -> [String] {
-        return CSwiftV.split("\n", string: string).filter { $0.isNotEmptyOrWhitespace }
+        return  CSwiftV.split(separator: "\n", string: string).filter { ($0 as String).isNotEmptyOrWhitespace }
```
Comment (dennisweissmann) [inline Sources/CSwiftV.swift:63]: is `($0 as String)` really necessary? It seems `$0` is already a String.

**Comment by cbarrerah** [comment]
Hi again,
It's possible it works without the  "as String" type cast, but i seem to remember the compiler complaining. As i was in a little bit of a rush, i made the necessary changes for it to compile without mindind about the details.

I have finished the exams, so i will have more time to review the code. I recently downloades the new Xcode beta, so i'll go over the changes and let you now if there's anything that has to change again.

Thanks for taking the time to review my changes ;)

File: Sources/CSwiftV.swift (hunk 1 above)
Comment (cbarrerah) [inline Sources/CSwiftV.swift:63]: Not really necessary, could work simply as the original $0.isNotEmptyOrWhitespace

**Comment by Daniel1of1** [comment]
Apologies for taking _so_ long to get round this, have been MIA I'm afraid.

Thanks so much for the contribution. I'm going to fix the merge issues now and get this in.

File: Sources/CSwiftV.swift (hunk 1 above)
Comment (Daniel1of1) [inline Sources/CSwiftV.swift:63]: fair point. No strong feeling either way from me.

**Pull request merged**